app/views/defence_requests/index.html.erb

```diff
@@ -53,12 +53,12 @@
         <td><%= def_req.detainee_first_name %></td>
         <td><%= def_req.gender %></td>
         <td><%= def_req.adult %></td>
-        <td><%= def_req.date_of_birth %></td>
         <td><%= def_req.appropriate_adult %></td>
         <td><%= def_req.custody_number %></td>
         <td><%= def_req.allegations %></td>
         <td><%= def_req.comments %></td>
         <td><%= def_req.time_of_arrival %></td>
+        <td><%= render partial: 'actions', locals: { def_req: def_req } %></td>
```

Comment:
You can omit the `:partial` and `:locals` keys as they are the defaults now, e.g:

``` erb
<%= render 'actions', def_req: def_req %>
```